sky/clouds/service_catalog/gcp_catalog.py

```diff
@@ -460,6 +460,7 @@ def get_image_id_from_tag(tag: str, region: Optional[str]) -> Optional[str]:
     return common.get_image_id_from_tag_impl(_image_df, tag, region)
 
 
-def validate_image_tag(tag: str, region: Optional[str]) -> bool:
+def is_image_tag_valid(tag: str, region: Optional[str]) -> bool:
     """Returns whether the image tag is valid."""
-    return common.is_image_tag_valid_impl(_image_df, tag, region)
+    del region  # Unused.
```

Comment:
Why unused? Thought we could pass it along?

Reply:
Our catalog for GCP images do not contain `Region` column, we can either handle it here or in the underlying function. I think it might be ok to do it here, as it is GCP specific. Wdyt?